### DISCUSSION TITLE
Add ergast laptimes data in seconds as a new row to fastf1 laps dataframe

### DISCUSSION THREAD

**Discussion by TracingInsights** [theOehrly/Fast-F1#304] [Ideas]
I noticed sometimes `LapTime` column data is missing and summing up `Sector1Time` `Sector2Time` and `Sector3Time` works to get the missing data, but this method also has some errors(missing data). We have 'LapTime' data in milliseconds available from ergast. It would be helpful to have it added to the fastf1 laps dataframe. This also fills up the missing lap 1 laptimes in some cases

**Comment by theOehrly** [comment]
Yes, absolutely. For this we first need better implementation of the Ergast API (#277) and then we need to add a robust strategy for merging the data from Ergast with the F1 timing data. If either of these services is unavailable, there should still be as much data returned as is possible.
Regarding the first lap time, there is already #293 in reference to a discussion about that. 
So basically, I am sort of working on all of that, but all together it's a non-trivial change to do it properly.
